docs/data/toolpad/connecting-to-datasources/connections.md

```diff
@@ -0,0 +1,86 @@
+# Connections
+
+<p class="description">A connection instance lets you reuse certain configuration details to create queries faster.</p>
+
+To create a new connection instance, press the **+** button in the Explorer menu on the left side of the interface:
+
+<img src="/static/toolpad/connection.png" alt="Add connection" width="196px" />
+
+## Connection types
+
+There are 2 kinds of data sources that do not require a connection instance:
+
+### Function
+
+<img src="/static/toolpad/connection-function-1.png" alt="Connection function" width="730px" />
```

Comment:
At a strict minimum, I would expect:

```suggestion
<img src="/static/toolpad/docs/connection-function-1.png" alt="Connection function" width="730px" />
```

since we also have a `/marketing` folder.